src/common/OneDSLoggerTelemetry/oneDSLogger.ts

```diff
@@ -194,11 +194,14 @@ export class OneDSLogger implements ITelemetryLogger{
         return instrumentationSettings;
       }
 
-	/// Trace info log
-	public traceInfo(eventName:string, eventInfo?:object, measurement?: object) {
-		const event = {
-			name: OneDSCollectorEventName.VSCODE_EVENT,
-			data: {
+    /// Trace info log
+    public async traceInfo(eventName: string, eventInfo?: object, measurement?: object) {
```

Comment:
You can't make traceInfo as async. Neither putting this kind of condition is generic. This method should be as simple as calling 1DSLogger.